README.md

```diff
@@ -46,17 +47,17 @@ go build grofer.go
 
 ### Docker
 
-1. Get docker image using
-   ```
+1.	Get docker image using
+
+```
    docker pull ghcr.io/pesos/grofer
-   ```
+```
+
+Note: This will use the `latest` image which is built from the main branch. For stable releases, you will need to use the version number as the tag. Example: `docker pull ghcr.io/pesos/grofer:1.3.0`
 
-   Note: This will use the `latest` image which is built from the main branch.
-   For stable releases, you will need to use the version number as the tag.
-   Example: `docker pull ghcr.io/pesos/grofer:1.3.0`
+1.	Run using
 
-1. Run using
-   ```bash
+```bash
```

Comment:
Same issue as #132 here ([this issue](https://github.com/pesos/grofer/pull/132#discussion_r693481183))

Reply:
Fixed